test/remote/gateways/remote_priority_test.rb

```diff
@@ -0,0 +1,352 @@
+require 'test_helper'
+
+class RemotePriorityTest < Test::Unit::TestCase
+  def setup
+    # Consumer API Key: Generated in MX Merchant for specific test merchant
+    # Consumer API Secret:= Generated in MX Merchant for specific test merchant
+
+    # run command below to run tests in debug (byebug)
+    # byebug -Itest test/unit/gateways/card_stream_test.rb
+    #
+    # bundle exec rake test:remote TEST=test/remote/gateways/remote_priority_test.rb
+    # ruby -Itest test/unit/gateways/priority_test.rb -n test_successful_void
+
+    # Run specific remote test
+    # ruby -Itest test/remote/gateways/remote_priority_test.rb -n test_fail_refund_already_refunded_purchase_response
+    @gateway = PriorityGateway.new(fixtures(:priority))
+
+    # purchase params success
+    @amount_purchase = 2.11
+    @credit_card_purchase_success = credit_card('4111111111111111', month: '01', year: '2029', first_name: 'Marcus', last_name: 'Rashford', verification_value: '999')
+
+    @option_spr = {
+      billing_address: address(),
+      avsStreet: '666',
+      avsZip: '55044',
+      tender_type: 'Card'
+    }
+
+    # purchase params fail inavalid card number
+    @credit_card_purchase_fail_invalid_number = credit_card('4111', month: '01', year: '2029', first_name: 'Marcus', last_name: 'Rashford', verification_value: '999')
+
+    # purchase params fail missing card number month
+    @credit_card_purchase_fail_missing_month = credit_card('4111111111111111', month: '', year: '2029', first_name: 'Marcus', last_name: 'Rashford', verification_value: '999')
+
+    # purchase params fail missing card verification number
+    @credit_card_purchase_fail_missing_verification = credit_card('4111111111111111', month: '01', year: '2029', first_name: 'Marcus', last_name: 'Rashford', verification_value: '')
+
+    # authorize params success
+    @amount_authorize = 7.99
```

Comment:
I think this is the same as the `@amount_purchase` in that it should be passed as an `int`.

Reply:
Removed